internal/store/custom_dns.go

```diff
@@ -56,7 +57,12 @@ func (s *Store) DisableCustomDNS(projectID string) (*atlasv2.AWSCustomDNSEnabled
 		customDNSSetting := &atlasv2.AWSCustomDNSEnabled{
 			Enabled: false,
 		}
-		result, _, err := s.clientv2.AWSClustersDNSApi.ToggleAWSCustomDNS(s.ctx, projectID).AWSCustomDNSEnabled(*customDNSSetting).Execute()
+		result, _, err := s.clientv2.AWSClustersDNSApi.ToggleAWSCustomDNS(s.ctx, projectID).
+			AWSCustomDNSEnabled(*customDNSSetting).Execute()
+		apiError := sdkv2.APIError(err)
+		if apiError != nil {
+			return nil, fmt.Errorf("%v", apiError)
```

Comment:
To minimize boilerplate we proposing returning error object directly in SDK in: https://github.com/mongodb/go-client-mongodb-atlas/pull/459